0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -48,7 +48,7 @@ A basic objective of the Interledger design is to tolerate a wide range of ledge
 
```

Comment:
This spec should no longer say "**This is a strawman proposal.** Choices made in this document are merely intended as a starting point." at the top